.agent/skills/scaffold_flow/generator.py

```
@@ -0,0 +1,117 @@
+
+        # B. Add to flows array
+        # Regex to find `flows: [...]` or `flows: array_variable`
+        # We assume `flows: [` pattern for simplicity based on project style
+        flows_pattern = r"(flows\s*:\s*\[)([^\]]*)(\])"
+        match = re.search(flows_pattern, content)
+
+        if match:
+            prefix, current_flows, suffix = match.groups()
+            if flow_name not in current_flows:
+                # Add comma if needed
+                new_flow_entry = f", {flow_name}" if current_flows.strip() else f"{flow_name}"
```

Ruff (0.14.14)

[error] Local variable `new_flow_entry` is assigned to but never used

Remove assignment to unused variable `new_flow_entry`

(F841)